cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -19,6 +20,9 @@
 {% if pillar['enable_deployments'] is defined -%}
   {% set enable_deployments = "--enable-deployment-controller=" + pillar['enable_deployments'] -%}
 {% endif -%}
+{% if grains.cloud in [ 'gce' ] and grains.cloud_config is defined -%}
```

Comment:
I presumed we only wanted this for gce/gke from reading discussion #7660. Alternatively we can just change the default value of the flag (and not touch salt) then add this to the release notes.